pacsman/filesystem_dev_client.py

```diff
@@ -0,0 +1,131 @@
+'''
+This filesystem "dummy" client should be used for testing in development
```

Comment:
The main reason I switched from "can be used" to "should be used" is that right now it would be pretty slow for large sets of data. There are no indexes or dicts set up; it just loops through every single dataset to find certain patients or studies. 

I agree with removing "dummy" as it's a little smarter than that. What do you think about keeping the name "filesystem dev client" for now?